README.md

```diff
@@ -92,6 +92,8 @@ Note:
 
 If you run into problems, first search the issues and discussions in this repository. If you don't find anything, you can come talk to our friendly and active developers in the Infinite Red Community Slack ([community.infinite.red](http://community.infinite.red)).
 
+If you still need help after reaching out to the proper channels, feel free to open a new GitHub issue via `npx ignite-cli issue "Unable to Ignite new app"` (as an example). This will help start writing your issue with the correct diagnostic information included.
```

Comment:
We really need an `ignite-cli` doc, documenting all commands. Not in this PR though, filed an issue #2291.